syntax = "proto2";

package hw1;

message type0 {
    required uint32 type = 1;
}
message type1 {
    required uint32 type = 1;
    required uint32 key=2;
    required string value=3;
}
message type2 {
    required uint32 type = 1;
}
message type3 {
    required uint32 type = 1;
    required uint32 key=2;
}
message type4 {
    required uint32 type = 1;
    required string value=2;
}
message type5 {
    required uint32 type = 1;
}
message type6 {
    required uint32 type = 1;
}
message type7 {
    required uint32 type = 1;
    required uint32 count=2;
}
message type8 {
    required uint32 type = 1;
    required string error=2;
}
message type9 {
    required uint32 type = 1;
    required string flag=2;
}